build_info/dbus.control

```diff
@@ -0,0 +1,31 @@
+Package: dbus
+Version: @DEB_DBUS_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Depends: dbus-bin (= @DEB_DBUS_V@), dbus-daemon (= @DEB_DBUS_V@), dbus-system-bus-common (= @DEB_DBUS_V@), libdbus-1-3 (= @DEB_DBUS_V@), firmware (>= 14.0) | libexpat1
+Suggests: default-dbus-session-bus | dbus-session-bus
+Provides: dbus-system-bus (= @DEB_DBUS_V@), default-dbus-system-bus
+Section: Administration
+Priority: standard
+Description: simple interprocess messaging system (system message bus)
+ D-Bus is a message bus, used for sending messages between applications.
+ Conceptually, it fits somewhere in between raw sockets and CORBA in
+ terms of complexity.
+ .
+ D-Bus supports broadcast messages, asynchronous messages (thus
+ decreasing latency), authentication, and more. It is designed to be
+ low-overhead; messages are sent using a binary protocol, not using
+ XML. D-Bus also supports a method call mapping for its messages, but
+ it is not required; this makes using the system quite simple.
+ .
+ It comes with several bindings, including GLib, Python, Qt and Java.
+ .
+ This package provides a fully-functional D-Bus system bus with activation
+ support, used for communication between system services, and depends on
+ most of the other components of the reference implementation of D-Bus.
+ .
+ To provide a complete D-Bus session bus, install one of the packages
+ that implement the dbus-session-bus virtual package, such as
+ dbus-user-session. The recommended implementation is indicated by
+ the default-dbus-session-bus virtual package.
+ 
```

Comment:
???? What's with this line

Reply:
For some reason GitHub wasn’t displaying shit properly on the website

Reply:
Lol